### PR TITLE
Fixed result for asset property queries

### PR DESCRIPTION
**Pull request by diericd**
Only one result was while multiple assets where queried.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v2.0.5
+
+### Bug
+
+- Only one result was while multiple assets where queried in the asset property query.
```
Comment (vdpoora) [inline CHANGELOG.md:7]: This is not a sentence.

I think you a word

File: docs/content/1_release_notes/_index.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ This is the compatibility matrix for version listed on this page. If you have do
 | <= v1.2.x                            | <= v6.3.x                            |
 | >= v2.x                              | > v6.3                               |
 
+## v2.0.5
+
+### Bug
+
+- Only one result was while multiple assets where queried in the asset property query.
```
Comment (vdpoora) [inline docs/content/1_release_notes/_index.md:28]: Not a sentence

**Comment by vdpoora** [comment]
the `handleAssetMeasurementQuery` function now has unused parameter seriesLimit. If it is no longer needed, remove it from the function signature, otherwise reimplement it

**Comment by diericd** [comment]
> the `handleAssetMeasurementQuery` function now has unused parameter seriesLimit. If it is no longer needed, remove it from the function signature, otherwise reimplement it


resovled

File: go.mod [hunk 3]
```diff
@@ -1,34 +1,36 @@
 module github.com/factrylabs/factry-historian-datasource.git
 
-go 1.22.1
+go 1.23
```
Comment (vdpoora) [inline go.mod:3]: Also update README to mention this. (Or remove the version specifier in the readme)

**Review by vdpoora — APPROVED** [review]
If readme is updated, it is fine

**Pull request merged**